ironfish/src/telemetry/telemetry.ts

```diff
@@ -114,9 +115,41 @@ export class Telemetry {
   private metricsLoop(): void {
     Assert.isNotNull(this.metrics)
 
+    const inboundTrafficFields: Field[] = NetworkMessageTypeList.flatMap((messageType) => {
+      const value = this.metrics?.p2p_InboundTrafficByMessage.get(messageType)?.rate5m
+      return value
+        ? [
+            {
+              name: 'inbound_traffic_' + NetworkMessageType[messageType].toLowerCase(),
+              type: 'float',
+              value: value,
+            },
+          ]
+        : []
+    })
```

Comment:
What is your thinking behind filtering out empty meters? Is this related to P2P connectedness?

Reply:
Just for the type system. For some reason the `Assert.isNotNull(this.metrics)` was not guaranteeing that `this.metrics` was not null inside the lambda function. Not sure why. I didn't really look into why but I can investigate a little

Reply:
found a better way to do this by just looping through the list of `Meter`s